x-pack/filebeat/input/http_endpoint/handler.go

```diff
@@ -177,7 +177,9 @@ func (h *handler) logRequest(r *http.Request, status int, respBody []byte) {
 			zap.ByteString("http.response.body.content", respBody),
 		)
 	}
-	httplog.LogRequest(h.reqLogger, r, extra...)
+	// Limit request logging body size to 10kiB.
+	const maxBodyLen = 10 * (1 << 10)
+	httplog.LogRequest(h.reqLogger, r, maxBodyLen, extra...)
```

Comment:
The question is whether this should be configurable or whether this is enough.

Reply:
Let's go with what you have. If it's too limiting we can add a new control on the wall later.

![y4emM6_ci14ycF1WqK7zw4PJ30uib7gQCnNGaxtXnjY-2232382363](https://github.com/elastic/beats/assets/4565752/014bcf18-d608-48b4-9505-0c96df5a4c22)